irohad/validation/impl/chain_validator_impl.cpp

```diff
@@ -18,11 +18,14 @@
 #include "validation/impl/chain_validator_impl.hpp"
```

Comment:
|`line 43-45`|
|---|
@{07fe7a37-a139-4532-aecf-e201522a92f4,Alexey} Where is peerSubset checking

Reply:
In peerSubset() function of SupermajorityChecker.